include/sway/tree/workspace.h

```diff
@@ -110,13 +110,16 @@ void workspace_unwrap_children(struct sway_workspace *ws,
 
 void workspace_detach(struct sway_workspace *workspace);
 
-void workspace_add_tiling(struct sway_workspace *workspace,
+struct sway_container *workspace_add_tiling(struct sway_workspace *workspace,
 		struct sway_container *con);
 
 void workspace_add_floating(struct sway_workspace *workspace,
 		struct sway_container *con);
 
-void workspace_insert_tiling(struct sway_workspace *workspace,
+void workspace_insert_tiling_direct(struct sway_workspace *workspace,
```

Comment:
We should have a comment here explaining the difference in nuance between `workspace_insert_tiling_direct` and `workspace_insert_tiling`. I only understand it since I've been following along this thread :)